config/application.rb

```diff
@@ -31,6 +31,10 @@
   Bundler.require(*Rails.groups, *groups)
 end
 
+DBQueryMatchers.configure do |config|
```

Comment:
this is for rspec only.

please put this exact clause in
`spec/support/db_query_matchers.rb`